assets/panel/debugger.properties

```diff
@@ -494,13 +494,30 @@ editor.editBreakpoint=Edit breakpoint
 
 # LOCALIZATION NOTE (editor.addConditionalBreakpoint): Editor gutter context
 # menu item for adding a breakpoint condition on a line.
-editor.addConditionalBreakpoint=Add conditional breakpoint
+editor.addConditionalBreakpoint=Add condition
```

Comment:
Sorry, but this shouldn't have passed review. String changes require new string IDs.

@jasonLaster Can we fix this ASAP? This just landed in m-c :-\